pkg/infradriver/utils_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package infradriver
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	k8sv1 "k8s.io/api/core/v1"
+)
+
+//key1=value1:NoSchedule
+func TestFormatData(t *testing.T) {
```

Comment:
test case too few.

value can be empty.

Reply:
I have added more ut.